sdk/core/core-rest-pipeline/test/node/pipeline.spec.ts

```diff
@@ -1,18 +1,20 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { assert } from "chai";
-import * as https from "https";
 import * as http from "https";
+import * as https from "https";
+
+import { proxyPolicy, proxyPolicyName } from "../../src/policies/proxyPolicy";
+import { tlsPolicy, tlsPolicyName } from "../../src/policies/tlsPolicy";
+
+import { HttpClient } from "../../src/interfaces";
 import { HttpsProxyAgent } from "https-proxy-agent";
-import sinon from "sinon";
-import { createPipelineFromOptions } from "../../src/createPipelineFromOptions";
+import { assert } from "chai";
```

Comment:
nit: 
```suggestion
import { assert } from "@azure/test-utils";
```

Reply:
This is a good point. This package doesn't depend yet on @azure/test-utils. So I think I'll make a separate PR to use it in all the tests in the package and keep this PR scoped to the tls policy